### ISSUE TITLE
Package name should be case insensitive

### ISSUE THREAD

**Issue by Clonkk** [nim-lang/atlas#72]
```
$ atlas init
$ atlas clone arraymancer
$ atlas use nimjl # Because it uses arraymancer as a dependency
```

This creates 2 repo arraymancer : "Arraymancer" and "arraymancer" (capital A) difference.

IMO that shouldn't be possible & package name should be case insensitive. I don"t want to have 3 different packages "Tools", "tools", "TOOLS" that are different things

**Comment by elcritch** [comment]
Good point, it should be lower-casing them. This is probably a bug then.

**Comment by Clonkk** [comment]
I think the issue is more general and related to how atlas resolve names
